src/lib/core.d.ts

```diff
@@ -1011,12 +1011,12 @@ interface Array<T> {
       * Combines two or more arrays.
       * @param items Additional items to add to the end of array1.
       */
-    concat<U extends T[]>(...items: U[]): T[];
+    concat<U extends T[]>(...items: (U | U[])[]): T[];

```

Comment:
I think this overload can be removed, as @ahejlsberg mentioned on https://github.com/Microsoft/TypeScript/issues/6594#issuecomment-174315415.